src/gui/dialogs/drop_down_menu.cpp

```diff
@@ -139,8 +139,7 @@ void drop_down_menu::pre_show(window& window)
 		find_widget<toggle_panel>(&new_row, "panel", false).set_tooltip(entry["tooltip"]);
 
 		if(entry.has_attribute("image")) {
-			image* img = new image();
-			img->set_definition("default");
+			image* img = build_single_widget_instance<image>("image");
```

Comment:
I don't think it matters that much, just seemed a little weird that only the definition was moved.